modules/exploits/multi/http/gitlab_file_read_rce.rb

```diff
@@ -554,11 +539,27 @@ def exploit
     secret_key_base = read_secret_key_base
 
     payload = build_payload
-    signed_cookie = sign_payload(secret_key_base, payload)
+    signed_cookie_value = sign_payload(secret_key_base, payload)
+
+    cj = HTTP::CookieJar.new
```

Comment:
`cookie_jar` is already available:

```suggestion
    cj = self.cookie_jar
```

Which actually makes this line redundant, as we can just use `cookie_jar` instead of the more cryptic `cj` 🕵️

Reply:
So with the API we're aiming for I think this ends up being:
```ruby
cookie_jar.clear
cookie_jar.set('experimentation_subject_id', signed_cookie_value)
```

Thining ahead and future proofing an API for a module were we don't have the luxury of clearing the entire cookie jar by itself, we could also use:
```ruby
cookie_jar = Exploit::Remote::HttpCookieJar.new
cookie_jar.set('experimentation_subject_id', signed_cookie_value)
```